hybrid/layout/pv_design_utils.py

```diff
@@ -210,7 +237,7 @@ def spe_power(spe_eff_level, spe_rad_level, spe_area) -> float:
     return spe_eff_level / 100 * spe_rad_level * spe_area
 
 
-def get_module_attribs(pvsam_model: pv.Pvsamv1) -> dict:
+def get_module_attribs(pvsam_model: pv_detailed.Pvsamv1) -> dict:
```

Comment:
Would be really nice to have a docstring here linking to the attributes in the PySAM documentation

Reply:
Done